extensions/wikia/ArticleVideo/scripts/ooyalaVideo.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * If we want to continue using ooyala player for "premium videos"
+ * we should merge this with ooyala handler (Ooyala.js from VideoHandlers dir)
+ */
+define('ooyalaVideo', ['jquery', 'wikia.window', 'wikia.loader'], function ($, window, loader) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "loader". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)